metabase/metabase
# Add basic filter modal

Epic https://github.com/metabase/metabase/issues/34112
Includes changes from:
- https://github.com/metabase/metabase/issues/35746
- https://github.com/metabase/metabase/issues/35882
- https://github.com/metabase/metabase/issues/36033
- https://github.com/metabase/metabase/issues/36034
- https://github.com/metabase/metabase/issues/36035
- https://github.com/metabase/metabase/issues/36036
- https://github.com/metabase/metabase/issues/36037
- https://github.com/metabase/metabase/issues/36037
- https://github.com/metabase/metabase/issues/36052

How to verify:
- Basic functionality should work. You should be able to click on "Filter" button in simple mode and apply filters in the modal.
- For category fields there would be no checkboxes like in `master`. There is a separate task for that. Same for segments, search, and column ordering.

<img width="1147" alt="Screenshot 2023-12-07 at 20 01 50" src="https://github.com/metabase/metabase/assets/8542534/6194fcba-30f0-4e14-a058-fbc79443a519">
